core/primitives/transaction_validity.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_PRIMITIVES_TRANSACTION_VALIDITY_HPP_
+#define KAGOME_CORE_PRIMITIVES_TRANSACTION_VALIDITY_HPP_
+
+#include <variant>
+#include <vector>
+
+namespace kagome::primitives {
+
+  namespace {
+    /// Priority for a transaction. Additive. Higher is better.
+    using TransactionPriority = uint64_t;
+
+    /// Minimum number of blocks a transaction will remain valid for.
+    /// `TransactionLongevity::max_value()` means "forever".
```

Comment:
Why not create an alias or rename max_value() to forever()? That seemingly would be more clear.

Reply:
Let's think about that when we will start using TransactionValidity structure. Otherwise we will leave it as unusable variable of which we will soon forget:)